assets/js/romo/base.js

```diff
@@ -182,10 +182,36 @@
   // elem handling
 
   Romo.prototype.toArray = function(elems) {
-    // converts a collection of elements (`$()`) to an array of nodes
+    // converts a collection of elements `$()` to an array of nodes
     return $.map(elems, function(node){ return node; })
   }
 
+  Romo.prototype.selectNext = function(elem, selector) {
+    // like `$().next()`, but takes a selector; returns first next elem that
+    // matches the given selector or an empty collection if non matches
+    var el = elem.next();
+    while(el.length) {
+      if (selector === undefined || el.is(selector)) {
+        return el;
+      }
+      el = el.next();
+    }
+    return el;
+  }
+
+  Romo.prototype.selectPrev = function(elem, selector) {
+    // like `$().prev()`, but takes a selector; returns first prev elem that
+    // matches the given selector or an empty collection if non matches
+    var el = elem.prev();
```

Comment:
@jcredding dang, I misread their docs and thought they didn't take selectors.  Oh well, the eventual goal is to not require jQuery so I'll leave this as is for now.